include/z64.h

```diff
@@ -253,8 +253,8 @@ typedef struct {
     struct {
         /* 0x0104 */ u32    swch;
         /* 0x0108 */ u32    tempSwch;
-        /* 0x010C */ u32    unk0;
-        /* 0x0110 */ u32    unk1;
+        /* 0x010C */ u32    unk_0;
+        /* 0x0110 */ u32    unk_1;
```

Comment:
These aren't named based on the offset and literally mean "unknown flags 1" and "unknown flags 2" since we don't know what they were intended for, so it might be confusing to have them as `unk_X`.
That being said, they could probably be named `unk` and `tempUnk` since they follow the same pattern as other flags with a temp field (see `Flags_GetUnknown` and such)

Reply:
Neither are saved in the save context so I don't think they should go by unk/tempUnk even if the pattern does fit
I'll revert to unk0 and unk1
and also remove "current scene" from the `Flags_*Unknown` functions since the flags arent saved nor loaded so aren't per-scene